citestwheel.Dockerfile

```
@@ -56,6 +56,10 @@ ENV PATH="/pyenv/versions/${PYTHON_VER}/bin/:$PATH"
 COPY --from=aws-cli /usr/local/aws-cli/ /usr/local/aws-cli/
 COPY --from=aws-cli /usr/local/bin/ /usr/local/bin/
 
+# Install rapids-dependency-file-generator
+RUN pyenv global ${PYTHON_VER} && python -m pip install rapids-dependency-file-generator \
```

```suggestion
RUN pyenv global ${PYTHON_VER} && python -m pip install 'rapids-dependency-file-generator>=1.14.0,<2.0.0a0' \
```

Let's put an upper bound on this, to avoid breaking CI if a 2.0 is ever released. As is done for the other CI images, like: 

https://github.com/rapidsai/ci-imgs/blob/c57ffbdccfbd4ace85986237b4b6d441624bfcde/ci-conda.Dockerfile#L134